streamparse/cli/submit.py

```diff
@@ -205,6 +205,17 @@ def submit_topology(name=None, env_name="prod", workers=2, ackers=2,
             if isinstance(inner_shell, ShellComponent):
                 if 'streamparse_run' in inner_shell.execution_command:
                     inner_shell.execution_command = streamparse_run_path
+                    
+    # Additional options
+    additional_options = env_config.get('options', None)
+    if additional_options and isinstance(additional_options, dict):
+        if isinstance(options, dict):
+            additional_options.update(options)
+        options = additional_options
+    if not workers:
+        workers = env_config.get('worker_count', 2)
```

Comment:
Also with
`workers = env_config.get('workers', workers)`
CLI args will be overwritten. So if defaults are 2, config.json would be 3 and CLI would be 4.
with proposed code it would stuck on 3, and not 4.